router/config.go

```diff
@@ -204,3 +204,16 @@ func (c *Config) LoadString(json_config string) error {
 	}
 	return c.verify()
 }
+
+type ConfigServiceConfig struct {
+	Port            string
+}
+
+func GetConfigServiceUrl() string {
+	config := ConfigServiceConfig{}
```

Comment:
There is definitely value in having concrete types when marshalling to/from JSON, since the marshaller is going to perform type checking at conversion time. By using a late cast .(string), you end up having to perform error checking in many different places after decoding.